api/src/utils/redact-header-cookies.ts

```diff
@@ -1,7 +1,9 @@
+import { redactText } from '../logger';
```

Comment:
Oh, another circular dependency 😄
Could we put that in `constants.ts` instead?

Reply:
updated!